src/igor/sub.go

```diff
@@ -82,6 +84,33 @@ func runSub(cmd *Command, args []string) {
 	var newSched []TimeSlice    // the new schedule
 	format := "2006-Jan-2-15:04"
 
+	// parse duration requested
+	var days int = 0
+	duration = 0
+	nanoseconds, err := time.ParseDuration(subT)
+	if err != nil {
+		// Check for a number of days in the argument
+		var dInd int
+		if dInd = strings.Index(subT, "d"); dInd >= 0 {
+			days, err = strconv.Atoi(subT[:dInd])
+			if err == nil {
+				duration = days * 24 * 60 //convert to minutes
+				subT = subT[dInd+1:]      //remove days from string
+				if subT != "" {           // capture any additional time
+					nanoseconds, err = time.ParseDuration(subT)
+				}
+			}
+		}
+		if err != nil {
+			log.Fatal("Unable to parse -t argument: %v\n", err)
+		}
+	}
+	fmt.Printf("duration: %v, nano: %v\n", duration, nanoseconds/time.Minute)
```

Comment:
Should probably `log.Debug` this.

Reply:
log.Fatal should catch this, and Fatal seems like the appropriate action here. Let's discuss this one when you are back.

Reply:
I meant for the `fmt.Printf`.